src/EnergyPlus/PluginManager.cc

```diff
@@ -110,17 +430,24 @@ namespace PluginManagement {
     }
 
 #if LINK_WITH_PYTHON == 1
-    std::string pythonStringForUsage() {
-        std::string sVersion = Py_GetVersion();
+    std::string pythonStringForUsage()
+    {
+        if (DataGlobals::eplusRunningViaAPI) {
+            return "Python Version not accessible during API calls";
```

Comment:
Return early to avoid attempting to make a Python function call during API runs.